src/common/RoutingList/hooks/useGoBack/useGoBack.js

```diff
@@ -0,0 +1,22 @@
+import { useCallback } from 'react';
+import {
+  useHistory,
+  useLocation,
+} from 'react-router-dom';
+
+const ORDER_LINE_URL = '/orders/lines/view';
+
+export const useGoBack = (poLineId) => {
```

Comment:
do you really need this hook? 

1. what is the problem to navigate to po line from routing list view?
2. the same for delete and create actions
3. for edit I believe we should navigate back to routing list view

this hook is useless, you don't need back, you need to specific route in different cases

Reply:
In most cases, it redirects to POL. However, we have such a component used with Titles in Receiving up. https://folio-org.atlassian.net/browse/UIREC-328